.github/FUNDING.yml

```diff
@@ -10,4 +10,8 @@ liberapay: # Replace with a single Liberapay username
 issuehunt: # Replace with a single IssueHunt username
 otechie: # Replace with a single Otechie username
 lfx_crowdfunding: # Replace with a single LFX Crowdfunding project-name e.g., cloud-foundry
-custom: ['https://distributeaid.org/donate', 'https://www.paypal.com/donate/?hosted_button_id=VCREWCLA6WTB4']
+custom:
+  [
+    "https://distributeaid.org/donate",
+    "https://www.paypal.com/donate/?hosted_button_id=VCREWCLA6WTB4",
+  ]
```

Comment:
**praise:** omg thanks so much for looking out and adding this!

**suggestion(non-blocking):** Can we include our US fiscal host for donations via card? `https://www.omprakash.org/global/distribute-aid/donate`

Reply:
I didn't add it, I just formatted it ;)

Reply:
Added the link!